### ISSUE TITLE
How to define an engine geometry that is to be used within medium to higher fidelity aerodynamic calculations?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DLR-SL/CPACS#135]
```
Occurs in xPath: 
/cpacs/vehicles/engines/engine/geometry

Problem/Question: 
How to define an engine geometry that is to be used within medium to higher 
fidelity aerodynamic calculations?

Comments: 
Use a similar approach as for the definition of a fuselage (and wing), i.e. let 
the user define actual cross-sectional profiles and connect these to generate 
actual engine aerodynamic shapes

daniel.boehnke@dlr.de
erwin.moerland@dlr.de
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 24 Jan 2012 at 5:18

**Comment by DLR-boeh-da** [comment]
We are currently developing a modeling for nacelle geometries that is based on a polar loft algorithm. Might become available in CPACS 2.3, maybe postponed.

**Comment by DLR-boeh-da** [comment]
The draft version will go live in CPACS 2.3 Certainly, there may be updates to the definition in upcoming versions. These include the definition of guidecurves. Awaiting input from Carsten & Tobias.

**Comment by DLR-LY** [comment]
The implementation in CPACS has to be postponed to CPACS 2.4 at the end of 2016.

**Comment by codingpoets** [comment]
A proposal was developed as part of the VicToria project and implemented in the _nacelle-geometry_ branch. It will be tested until the end of the project and then be considered for integration in the main development branch.

**Comment by MarAlder** [comment]
merged and closed by #581
